modules/troubleshooting-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins-reference.adoc
+
+:_content-type: REFERENCE
+[id="troubleshooting-dynamic-plug-in_{context}"]
+= Troubleshooting your dynamic plug-in
+
+Refer to this list of troubleshooting tips if you run into issues loading your plug-in. 
+
+* You have enabled your plug-in in the console Operator configuration. Run the following command to check to check if you have enabled your plug-in is in the console Operator configuration and your plug-in name is the output.
```

Comment:
This don't sound right: 

> check if you have enabled your plug-in is in the console Operator configuration

_if you have enabled your plug-in and that it is in the console Operator configuration_ ??

Reply:
> and your plug-in name is the output.

This might read better as it's own sentence.

Reply:
s/You have enabled/Verify that you have enabled ?? The latter sounds for like a Troubleshooting tip to me.